packages/common/src/api/fetchData.ts

```diff
@@ -90,19 +90,25 @@ export const fetchData: FetchData = <T, D = any>(
       if (error.response) {
         const status = error.response.status
         const message = error.response.statusText
+        const data = error.response.data
```

Comment:
I know it means adding logic in the fetcher that could potentially for some endpoint not work like that. But I'd say it's pretty safe for most cases to have:

```ts
        const data = error.response.data as {message?: string; error?: string}
        const message =
          data?.message ?? data?.error ?? error.response.statusText
   ```

Reply:
Yap, I was bugging about the same thing, still there are some errors that might contains a response data detailing the problem, therefore if we can't merge (message + data), @banklesss I'd like at least to have the `data` renamed to `responseData`. So at least the semantic is better, and the client has a clear way to inspect the response body in case of error. Like 400, Bad Request (missing fields, or invalid fields will come in the body)